habitat_baselines/config/default.py

```diff
@@ -14,92 +14,83 @@
 DEFAULT_CONFIG_DIR = "configs/"
 CONFIG_FILE_SEPARATOR = ","
 # -----------------------------------------------------------------------------
-# Config definition
+# EXPERIMENT CONFIG
 # -----------------------------------------------------------------------------
 _C = CN()
 _C.BASE_TASK_CONFIG_PATH = "configs/tasks/pointnav.yaml"
 _C.TASK_CONFIG = CN()  # task_config will be stored as a config node
 _C.CMD_TRAILING_OPTS = ""  # store command line options"
+_C.TRAINER_NAME = "ppo"
+_C.ENV_NAME = "NavRLEnv"
+_C.SIMULATOR_GPU_ID = 0
```

Comment:
We have setups when we use multiple  GPU ids for simulators. Maybe we want to be more conservative what to put in root node. `_C.ENV_NAME` is user for RL trainers mostly.